test/feature/support/helpers.coffee

```diff
@@ -19,12 +19,21 @@ class Helpers
       app.use exprestive(#{optionsStr})
       app.listen #{@port}
 
+      # Route to evaluate functions from tests
+      app.get '/eval/:strToEval', (req, res) ->
```

Comment:
Do you mean putting this in an exprestive controller? This would mean creating a routes file for every test run which isn't practical because some test runs define their own custom routes file